CONTRIBUTING.md

```diff
@@ -0,0 +1,57 @@
+This document is a guide for contributions to the WGPU project.
+
+## Welcome!
+
+First of all, welcome to the WGPU community! 👋 We're glad you want to
+contribute. If you are unfamiliar with the WGPU community as an organization, we
+recommend you read [`GOVERNANCE.md`] for an overview of how it works.
+
+[`GOVERNANCE.md`]: ./GOVERNANCE.md
+
+## Talking to other humans in the WGPU project
+
+The WGPU project has multiple ways to engage:
+
+- [GitHub discussions]: TODO: not sure when to use this over Matrix and issues.
+  TBH.
+- The Matrix channel [`wgpu:matrix.org`](https://matrix.to/#/#wgpu:matrix.org)
+  is dedicated to informal chat about contributions the project. It is
+  particularly useful for:
+  - …saying hello, and introducing yourself.
+  - …validating contributions (i.e., determining if they'll be accepted,
+    ensuring your approach is correct, making sure you aren't wasting effort,
+    etc.)
+  - …setting expectations for contributions 
```

Comment:
Should probably add:
```
 - Asking questions.
```

What's with the leading `...` here? Looks a bit unusual to me.